packages/jest-circus/src/__tests__/hooksError.test.ts

```diff
@@ -3,17 +3,13 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- *
- * @flow
  */
 
-'use strict';
-
-const circus = require('../index.js');
+import circus from '../';
 
 describe.each([['beforeEach'], ['beforeAll'], ['afterEach'], ['afterAll']])(
   '%s hooks error throwing',
-  fn => {
+  (fn: 'beforeEach' | 'beforeAll' | 'afterEach' | 'afterAll') => {
```

Comment:
would be cool to type `jest-each` so that it would be inferred.

/cc @mattphillips